Configurations.md

```diff
@@ -2692,17 +2755,6 @@ By default this option is set as a percentage of [`max_width`](#max_width) provi
 
 See also [`max_width`](#max_width) and [`use_small_heuristics`](#use_small_heuristics)
 
-## `style_edition`
-
-Controls the edition of the [Rust Style Guide] to use for formatting ([RFC 3338])
-
-- **Default value**: `"2015"`
-- **Possible values**: `"2015"`, `"2018"`, `"2021"`, `"2024"` (unstable variant)
-- **Stable**: No
-
-[Rust Style Guide]: https://doc.rust-lang.org/nightly/style-guide/
-[RFC 3338]: https://rust-lang.github.io/rfcs/3338-style-evolution.html
-
```

Comment:
Please revert this change. I don't want to make any changes to other config documentation in this PR.

Reply:
[this change](https://github.com/rust-lang/rustfmt/pull/6066/files/fede8d0de367185aa403780b595ca76d10282a52#r1795880128) you mentioned here should actually be here. will fix it.